api/discovery/v1/foreigncluster_types.go

```diff
@@ -52,11 +53,12 @@ type ForeignClusterStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	Joined             bool                `json:"joined"`
-	PeeringRequestName string              `json:"peering-request-name"`
-	CaDataRef          *v1.ObjectReference `json:"caDataRef,omitempty"`
-	Advertisement      *v1.ObjectReference `json:"advertisement,omitempty"`
-	Ttl                int                 `json:"ttl,omitempty"`
+	Joined                   bool                `json:"joined"`
```

Comment:
Proposal: Maybe we can split between the incoming and the outgoing part to avoid confusion between parameters affecting one or the other direction? Does it seem useful?

Reply:
This field will become an enum, it will be "None, Incoming, Outgoing, Bidirectional"